wwwroot/data/regionids/region_map-FID_LGA_2013_AUST_STE_NAME11.json

```diff
@@ -0,0 +1 @@
estern Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Australian Capital Territory","Australian Capital Territory","Other Territories","Other Territories"]}
```

Comment:
Oh wait, I see you have `+makeJSON "region_map:FID_LGA_2013_AUST" STE_NAME11 '"' # yes, 11...` up above - can you explain that?